mikupad.html

```diff
@@ -160,8 +164,8 @@
 }
 @media (min-width: calc(40.5em + 250px)) {

```

Comment:
You also need to change this media rule and the one above, this should be the code that you need to use:
```css
@media (min-width: calc(calc(var(--prompt-container-width) * 1.125rem) + 250px))
```